data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,585 @@ mission "Heliarchs Buy Jump Drive Mission"
+
+
+mission "Coalition: Struggling Author 4"
+	name "Kimek Publishing Company"
+	description "Anna found a Kimek publishing company who is willing to take in Anna Fangvin to get her book published. Bring Anna Fangvin to <destination>."
+	source "Ki Patek Ka"
+	destination "Inmost Blue"
+	blocked "You do not have enough bunk space for the Heliarch agent and Anna Fangvin. Come back here when you have two bunks free."
+	passengers 2
+	to offer
+		has "Coalition: Struggling Author 3: done"
+	on offer
+		conversation
+			`The spaceport of Ki Patek Ka is vast. You decide to search for Anna, who has chosen to settle in the cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`
+			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <destination>," explains Anna, once again enthusiastic.`
+			choice
+				`	"Great!"`
+					goto go
+				`	"Let's go!"`
+					goto go
+				`	"What's the publishing company called?"`
+			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`
+			label go
+			`	Anna heads to your ship, and you plot a course to <destination>. Just before you launch, the Heliarch hops in.`
+				launch
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive.' She hastily opens the hatch as soon as it unlocks.`
+			`	Anna stops, waiting for you to catch up. "In the message they sent me, they included instructions on how to reach their offices," she explains. "Let's go!"`
+			`	You begin following Anna to the publisher's offices. You walk through the spaceport and its endless rows of shops and streets, eventually approaching what looks like a train station. Anna looks at the message again and tells you which platform to go to and which train to take. After a few minutes the train arrives. You and Anna board the train and travel to a neighborhood, with a large white building dominating the skyline.`
+			`	Once the train stops, you get off onto the platform and out of the station. You walk around the building and find an entrance, busy with many Kimek and the occasional Saryd and Arach entering and exiting. Anna looks for a front desk and directs you to walk to a Kimek who looks like a receptionist.`
+			choice
+				`	"Hello, do you happen to know where Priyepichia Publishing is?"`
+				`	"We're looking for a publishing company called Priyepichia Publishing. Could you provide directions?"`
+			`	"On the 16th floor, Priyepichia Publishing is. Over there, the elevators are," says the receptionist's translator.`
```

Comment:
Think this chunk is a _bit_ too long. Maybe add a choice to exit the elevator?
![image](https://github.com/user-attachments/assets/979c4fdd-ba14-40a0-9589-3e5ed370dc79)